### ISSUE TITLE
The touchpad cannot be compatible with tapping and multi-finger gestures

### ISSUE THREAD

**Issue by GITHUBNULLING** [badstorm/surface-pro-7-opencore#8]
Hello, SP7 i5, Big Sur 11.1, OpenCore 0.65, Prior to this, it was impossible to tap and click. In fact, the touchpad buttons are also abnormal, seemingly mixed with functions such as right-click and space, but multi-finger gestures are normal, which can achieve two-finger or scrolling and swiping. Three-finger gesture.
After tapping to turn on, the touchpad cannot recognize multi-finger gestures, only a single finger, scrolling and swiping cannot be realized.
Not compatible.
Is your touchpad completely normal?
(From Google Translate)

**Comment by badstorm** [comment]
As write in the README, for the touchpad you need to run this commands in the terminal:

```
sudo defaults write NSGlobalDomain com.apple.trackpad.forceClick -bool false
sudo defaults write com.apple.AppleMultitouchTrackpad ForceSuppressed -bool true
sudo defaults write com.apple.AppleMultitouchTrackpad ActuateDetents -bool false

sudo defaults write com.apple.AppleMultitouchTrackpad Clicking -bool true 
sudo defaults write com.apple.driver.AppleBluetoothMultitouch.trackpad Clicking -bool true 
sudo defaults -currentHost write NSGlobalDomain com.apple.mouse.tapBehavior -int 1 
sudo defaults write NSGlobalDomain com.apple.mouse.tapBehavior -int 1

sudo defaults write com.apple.AppleMultitouchTrackpad Dragging -bool true 
sudo defaults write com.apple.driver.AppleBluetoothMultitouch.trackpad Dragging -bool true

# REBOOT
```

**Comment by GITHUBNULLING** [comment]
The command is executed, but does not take effect.

**Comment by badstorm** [comment]
Actually I not found any other solution and in the most case this works.

You can just try to run this two commands without _sudo_

```
defaults write com.apple.AppleMultitouchTrackpad Clicking -bool true 

defaults write com.apple.AppleMultitouchTrackpad Dragging -bool true 

```
But if works, remember to run them on all users present in the system.

**Comment by GITHUBNULLING** [comment]
I found part of the reason. VoodooI2C.kext is not sorted correctly, which makes it impossible to use tap and click.  Your config.plist is like this.
![2021-01-15 15 04 25](https://user-images.githubusercontent.com/3806161/104693573-dea9e180-5744-11eb-813b-f1fbb2b615d9.png)

I can only use multi-finger gestures. After I modified it, 
![2021-01-15 15 03 40](https://user-images.githubusercontent.com/3806161/104693597-e8334980-5744-11eb-9ec3-7a6fdca60a37.png)

tap and click can be used, but multi-finger Gestures cannot be used, so they are still not compatible.
